pkcs12.go

```diff
@@ -207,6 +209,9 @@ func convertAttribute(attribute *pkcs12Attribute) (key, value string, err error)
 		// This key is chosen to match OpenSSL.
 		key = "Microsoft CSP Name"
 		isString = true
+	case attribute.Id.Equal(oidJavaTrustStore):
+		key = oidJavaTrustStore.String()
+		isString = true
```

Comment:
Considering `ToPEM` is buggy and deprecated, I would prefer not to add any new functionality to it.  Could you remove this change?

Reply:
Done, this is a leftover from my manual testing decoding keystores to check the attribute contents.